akka-actor/src/main/scala/akka/NotUsed.scala

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (C) 2009-2016 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka
+
+/**
+ * Type of value is not intended to be used. Is more clear than Unit and available
+ * both from Scala and Java (which ```Unit``` is not).
```

Comment:
“This type is used in generic type signatures wherever the actual value is of no importance. It is a combination of Scala’s Unit and Java’s Void, which both have different issues when used from the other language. An example use-case is the materialized value of an Akka Stream for cases where no result shall be returned from materialization.”